sklearn/metrics/tests/test_ranking.py

```diff
@@ -467,7 +467,7 @@ def test_precision_recall_curve_pos_label():
     assert_array_almost_equal(r, r2)
     assert_array_almost_equal(thresholds, thresholds2)
     assert_equal(p.size, r.size)
-    assert_equal(p.size, thresholds.size + 1)
```

Comment:
We probably want. It would be odd to have different shapes depending on the data. This might have been the reason for the original behavior?